### PR TITLE
[1LP][RFR]workaround for provisioning tabs filling

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================

__Fixing__ tests which were filling provisioning tabs. right now each 2nd test was failing because of 
some JS issues. https://projects.engineering.redhat.com/browse/RHCFQE-4901 
{{pytest: cfme/tests/infrastructure/test_provisioning_dialog.py --long-running --use-provider complete}}

### CODE REVIEW

**Comment by lkhomenk** [comment]
PRT was passed with --complete with more than 90% for 5.7.or 5.8
Issues are not related to this PR.

**Comment by lkhomenk** [comment]
@jkrocil @psav - actually this is a workaround I was talking.

File: cfme/common/vm_views.py [hunk 1]
```diff
@@ -15,6 +15,12 @@
     BaseQuadIconEntity, BaseTileIconEntity, JSBaseEntity, BaseNonInteractiveEntitiesView)
 
 
+class ProvisionTab(Tab):
+    """Workaround for testing purposes only"""
```
Comment (mshriver) [inline cfme/common/vm_views.py]: If this is linked to a BZ, can you include the # here?

Comment (mshriver) [reply cfme/common/vm_views.py]: @lkhomenk shared a message thread with me regarding this behavior, no applicable BZ for now.

**Pull request merged**